modulus/distributed/autograd.py

```diff
@@ -50,7 +50,7 @@ def forward(
         dim: int = 0,
         use_fp32: bool = True,
         group: Optional[dist.ProcessGroup] = None,
-    ) -> torch.Tensor:
+    ) -> torch.Tensor: # pragma: no cover
```

Comment:
I understand why the pragma is needed here. I created a new issue to try and fix this. Can you add any other details I might have missed to this issue? https://github.com/NVIDIA/modulus/issues/165

Reply:
No, seems good to me, thanks for opening the issue 👍🏽